mycli/packages/special/iocommands.py

```diff
@@ -115,7 +115,7 @@ def get_editor_query(sql):
     # The reason we can't simply do .strip('\e') is that it strips characters,
     # not a substring. So it'll strip "e" in the end of the sql also!
     # Ex: "select * from style\e" -> "select * from styl".
-    pattern = re.compile('(^\\\e|\\\e$)')
+    pattern = re.compile(r'(^\\e|\\e$)')
```

Comment:
If this becomes a raw string wouldn't that mean only one `\` is required to escape the `\e`  or am I mistaken?

Reply:
If it was `print(r'\e')` then yes, only one, but inside a regular expression Python would attempt to interpret `\e` as a (nonexistent) regular-expression escape.  I keep that straight in my head by thinking: if `\` has special semantics in a regex, then an escape is needed to quote the meaningful backslash.  (But actually it's unfortunate design that `\` has two meanings in regular expressions.)

Reply:
:+1: